index.html

```diff
@@ -187,14 +193,25 @@ <h3>
               method="POST"
               class="contact-form"
             >
-              <input
-                type="text"
-                name="full-name"
-                id="name"
-                placeholder="Full name"
-                maxlength="30"
-                required
-              />
+              <div class="names">
+                <input
+                  type="text"
+                  name="full-name"
+                  id="name"
+                  placeholder="Name"
+                  maxlength="30"
+                  required
+                />
+                <input
+                  type="text"
+                  name="last-name"
+                  id="last-name"
+                  placeholder="Last name"
+                  maxlength="30"
+                  required
+                />
```

Comment:
- [ ] As mentioned in the project requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/213203162-488a06f0-2aa9-450f-a9e9-20be5e7c56f1.png)

How about we only use one input field for full name just like we have in the mobile version, this will make the forms more user friendly too :+1: